src/ripple/app/tests/MultiSign.test.cpp

```diff
@@ -621,6 +621,39 @@ class MultiSign_test : public beast::unit_test::suite
         env(regkey (alice, disabled), sig(alie));
     }
 
+    // Verify that the first regular key can be made for free using the
+    // master key, but not when multisigning.
+    void test_regKey()
+    {
+        using namespace jtx;
+        Env env(*this);
+        Account const alice {"alice", KeyType::secp256k1};
+        env.fund(XRP(1000), alice);
+
+        // Give alice a regular key with a zero fee.  Should succeed.  Once.
+        Account const alie {"alie", KeyType::ed25519};
```

Comment:
Right.  I got in the habit of varying the key types in the multisign tests because the new key type was introduced in the middle of multisign development.  I wanted to have some confidence that things would continue to work with heterogeneous key types.  I'm not worried about specific key types here, I'm just continuing the model that's used elsewhere in the file.  As you say, it doesn't hurt.